configure.ac

```diff
@@ -24,7 +24,7 @@ AC_MSG_RESULT([$GCC_VERSION])
 AX_COMPARE_VERSION([$GCC_VERSION],[ge],['6.0.0'],[],[AC_MSG_ERROR([At least GCC version 6 is required])])
 AC_DEFINE_UNQUOTED([GCC_PATH], ["$CC"], [GCC path (for compiling)])
 AC_SUBST(GCC_PATH, $CC)
-AX_LLVM([4.0],[15.0.1],[all])
+AX_LLVM([4.0],[15.0.6],[all])
```

Comment:
I think we need to revert this until we fix all the bugs in #333.

Reply:
No it works with LLVM up to 16 from the git on ubuntu 18
Beyond FMA fp-contract starting at llvm14, the remaining bug in  https://github.com/verificarlo/verificarlo/issues/333 relates to the default packages in ubunutu 22.04.
So I do not see the pb of going to 15.0.6

Reply:
Ok !